### ISSUE TITLE
etcd containers stuck in "Initializing" state when 2 hosts out of 5 are turned off[scale policy 3]

### ISSUE THREAD

**Issue by soumyalj** [rancher/rancher#5390]
**Rancher Version:**: 1.1.0

**Docker Version:**1.10.3

**OS and where are the hosts located? (cloud, bare metal, etc):**: Ubuntu, Cloud

**Environment Type: (Cattle/Kubernetes/Swarm/Mesos)**: Kubernetes

**Steps to Reproduce:**
1. Create a Kubernetes environment with 5 hosts(Scale policy is still 3)
2. Kubernetes must be functional, adding pods must be successful
3. Turn off two hosts.

etcd fails to start successfully on the other 2 hosts. It is stuck in initalizing state [cannot maintain quorum]

```
7/11/2016 11:19:18 AM+++ for i in '$(seq 1 $SCALE)'
7/11/2016 11:19:18 AM+++ giddyup probe http://Kubernetes_etcd_1:2379/health
7/11/2016 11:19:18 AM+++ '[' 1 == 0 ']'
7/11/2016 11:19:18 AM+++ for i in '$(seq 1 $SCALE)'
7/11/2016 11:19:18 AM+++ giddyup probe http://Kubernetes_etcd_2:2379/health
7/11/2016 11:19:18 AM+++ '[' 1 == 0 ']'
7/11/2016 11:19:18 AM+++ for i in '$(seq 1 $SCALE)'
7/11/2016 11:19:18 AM+++ giddyup probe http://Kubernetes_etcd_3:2379/health
7/11/2016 11:19:18 AM+++ '[' 1 == 0 ']'
7/11/2016 11:19:18 AM+++ '[' 0 '!=' 0 ']'
7/11/2016 11:19:18 AM+++ sleep 1
7/11/2016 11:19:19 AM+++ '[' 0 == 0 ']'
7/11/2016 11:19:19 AM+++ echo No etcd nodes available
7/11/2016 11:19:19 AM++ '[' 'No etcd nodes available' '!=' RUNNING ']'
7/11/2016 11:19:19 AM++ '[' 1 == 1 ']'
7/11/2016 11:19:19 AM++ bootstrap_node
7/11/2016 11:19:19 AM++ echo Waiting for discovery node to become healthy
7/11/2016 11:19:19 AM++ giddyup probe http://discovery:6666/health --loop --min 1s --max 60s --backoff 1.1
7/11/2016 11:19:19 AMWaiting for discovery node to become healthy
7/11/2016 11:19:19 AMGet http://discovery:6666/health: dial tcp: lookup discovery on 169.254.169.250:53: no such host
7/11/2016 11:19:20 AMGet http://discovery:6666/health: dial tcp: lookup discovery on 169.254.169.250:53: no such host
7/11/2016 11:19:21 AMGet http://discovery:6666/health: dial tcp: lookup discovery on 169.254.169.250:53: no such host
7/11/2016 11:19:23 AMGet http://discovery:6666/health: dial tcp: lookup discovery on 169.254.169.250:53: no such host
7/11/2016 11:19:24 AMGet http://discovery:6666/health: dial tcp: lookup discovery on 169.254.169.250:53: no such host
7/11/2016 11:19:25 AMGet http://discovery:6666/health: dial tcp: lookup discovery on 169.254.169.250:53: no such host
7/11/2016 11:19:27 AMGet http://discovery:6666/health: dial tcp: lookup discovery on 169.254.169.250:53: no such host
7/11/2016 11:19:29 AMGet http://discovery:6666/health: dial tcp: lookup discovery on 169.254.169.250:53: no such host
7/11/2016 11:19:31 AMGet http://discovery:6666/health: dial tcp: lookup discovery on 169.254.169.250:53: no such host
```

**Comment by soumyalj** [comment]
The above errors were seen in bug https://github.com/rancher/rancher/issues/5316

**Comment by LLParse** [comment]
Solved in Rancher `v1.1.1` and latest k8s template
